assets/css/_mixins.scss

```diff
@@ -0,0 +1,25 @@
+@import 'variables';
+
+@mixin flex($direction: row, $wrap: nowrap, $justify: space-between, $align: stretch, $gap: 5px) {
+    display: flex;
+    flex-direction: $direction;
+    flex-wrap: $wrap;
+    justify-content: $justify;
+    align-items: $align;
+    gap: $gap;
+}
+@mixin grid($columns: 12, $gutter: 1rem, $breakpoints: $breakpoints) {
```

Comment:
why do we need mixins like these (flex/grid)? can the same result be achieved in other ways?

Reply:
Can you please suggest any ?

Reply:
I see! I think we might not need to change anything for now 👍

Reply:
Ok. Bhai. Thanks